specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2024-03-01-preview/examples/GetDiagnostics.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
```

Comment:
>parameters [](http://example.com/codeflow?start=3&length=10)

nit consider renaming the example to ListDiagnostics

Reply:
Updated to ListDiagnostics